com.blitterstudio.amiberry.yml

```diff
@@ -0,0 +1,116 @@
+app-id: com.blitterstudio.amiberry
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: amiberry.sh
+rename-icon: amiberry
+copy-icon: true
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  # Wayland access
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Needs to talk to the network:
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: libserialport
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.a
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/raw/master/external/libserialport/libserialport-0.1.1.tar.gz
```

Comment:
1. you should never reference a file on a git branch.
2. why don't you get the tarball from upstream?

Reply:
Upstream was broken a few days ago, the URL was unreachable. I moved the archive in my repo as a failsafe, in case it's gone from upstream

Reply:
at minimum fix 1.

But tarball from a git repo, that's bad practice. git isn't made for that.

Reply:
I found a read-only mirror on Github, so I've switched to that.